KerbalStuff/app.py

```diff
@@ -101,10 +108,32 @@ def handle_500(e: Exception) -> Tuple[str, int]:
 
 
 @app.errorhandler(404)
-def handle_404(e: Exception) -> Tuple[str, int]:
+def handle_404(e: Exception) -> Union[Tuple[str, int], Response, werkzeug.wrappers.Response]:
+    path = request.path
+    if path.startswith('/api/'):
+        return handle_api_exception(e)
     return render_template("not_found.html"), 404
 
 
+def handle_api_exception(e: Exception) -> Union[Response, werkzeug.wrappers.Response]:
```

Comment:
What do you think about moving this function to `api.py`? It would not be called from there, but that way the API's code would be more centralized and easy to find.

Reply:
Yep, makes sense